### DISCUSSION TITLE
Self-hosted instance not registering events

### DISCUSSION THREAD

**Discussion by shawnmcclelland** [plausible/analytics#475] [Self-Hosted Support] [answered]
First time setting up a Plausible self-hosted instance and so far I've got everything set up on a DO droplet but there's no events being passed from my site to the analytics droplet. Main site is hosted on another droplet and everything is being routed through Cloudflare but dashboard just shows "Waiting for first page view". The only modification I made was using Cloudflare's origin server certs for SSL as certbot was failing using the provided reverse proxy from the hosting repository.

Below is the /etc/nginx/sites-available config if that helps. Looking for any insight here from the experts!

```
server {
	# replace example.com with your domain name
	server_name plausible.website.com;

	listen 80;
	listen [::]:80;

	location / {
		proxy_pass http://127.0.0.1:8000;
		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	}
}

server {

	# SSL configuration

	listen 443 ssl http2;
	listen [::]:443 ssl http2;
	ssl_certificate         /etc/ssl/cert.pem;
	ssl_certificate_key     /etc/ssl/key.pem;
	ssl_client_certificate  /etc/ssl/cloudflare.crt;
	ssl_verify_client on;

	server_name plausible.website.com;

	location / {
			proxy_pass http://127.0.0.1:8000;
			proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	}
}
```

**Comment by metmarkosaric** [comment, marked as answer]
Thanks @shawnmcclelland! Are you by any chance running Plausible on a local host? Our script automatically disables itself on localhost.

**Reply by shawnmcclelland** [reply]
No localhost. I removed it from my local dev environment of Ghost so it wasn't skewing anything. Good to know it gets disabled on localhost though. Everything's running off DO servers spread across two droplets for my set up.

**Reply by shawnmcclelland** [reply]
@metmarkosaric thanks for the help! Seems the issue was between the keyboard and chair (me). I restored from a fresh Ubuntu image, fixed some Cloudflare stuff that was interfering with Certbot and gave the Ghost instance a reboot. It seems something was cached since my site was private Plausible wasn't seeing the events as page views until I blew away the website data and logged back in. I've now got a dashboard and am seeing events again :) cheers!

**Reply by metmarkosaric** [reply]
good to hear! enjoy Plausible!

**Comment by KristerV** [comment]
Is it possible to close threads that are solved?

**Reply by metmarkosaric** [reply]
not possible to close but i've just marked this one as "answered"
